common/src/main/java/com/wynntils/wc/utils/ActionBarManager.java

```diff
@@ -41,7 +43,7 @@ public static void onActionBarUpdate(ChatReceivedEvent e) {
         maxMana = Integer.parseInt(matcher.group(5));
 
         String centerText = matcher.group(3);
```

Comment:
To do this really properly, I think we should have the ActionBarManager send out a wc event, like `ActionBarMessageUpdateEvent` with the group(3) content, and have the feature listen for it. That way we can make sure no "manager" depends on a "feature".

Reply:
I was also thinking about something like this but didn't implement it. It is done now.